---
layout: page
title: Research
permalink: /research/
---
####Chemical Approaches to Deciphering and Controlling Signal Transduction Pathways

**Identification of the Direct Substrates of Every Protein Kinase:**  The search for the complete set of all protein kinase substrates is a major goal of many laboratories.  It is estimated that one third of the proteome is phosphorylated making the tracing of the substrates of >500 kinases extremely challenging.  To address this problem we have devised a chemical method for tagging the direct substrates of any protein kinase using a [-S] labeled ATP analog, N6-(benzyl)ATP.  The ATP analog, N6-(benzyl)ATP is a poor substrate of wild-type protein kinases, but is efficiently accepted by any kinase of interest by virtue of a mutation which enlarges ATP binding site to accommodate the N6-benzyl substituent.  Identification of the thiophosphate labeled proteins via chemical capture allows for hundreds of novel substrates of over 50 widely divergent kinases, such as v-Src, CDK2, JNK, Cdc28, Erk2, Srb10, and kin28. The ability to directly affinity purify substrates of any kinase in the   will allow for the complete mapping of any kinase pathway in a cell and development of a complete picture of the complex networks of kinase signal transduction pathways.  It is our long term goal to identify all the direct substrates of each kinase in the human genome using these chemical tools.  Approaches for introducing these ATP analogs directly into intact cells are also being developed-allowing for the direct labeling of substrates in their undisturbed cellular compartments.

**Protein Kinase Inhibitors:**  We have developed a powerful chemical genetic method for the generation of target-specific inhibitors of any protein kinase in the genome.  This strategy utilizes a functionally silent active site mutation to sensitize a target kinase to inhibition by a small molecule that does not inhibit wild-type kinases.  Tyrosine and serine/threonine kinases are equally amenable to the drug-sensitization approach, which has been used to generate selective inhibitors of mutant Src family kinases, Abl family kinases, cyclin-dependent kinases (CDKs), mitogen-activated kinases (MAPKs), p21-activated kinases (PAKs), Ca2+/calmodulin-dependent kinases (CAMKs), and over 50 other protein kinases.  The ability to generate the very first inhibitors of many diverse protein kinases has allowed for the discovery of fundamentally new roles of kinases in transcription, cell cycle, cell-fate determination, the unfolded protein response, oncogenic transformation, and many others.**Lipid kinase inhibitors for the treatment of cancer:**  Phosphatidylinositol 3-kinases are activated by a wide range of cell surface receptors to generate the lipid second messengers phosphatidylinositol 3,4-bisphosphate (PIP2) and phosphatidylinositol 3,4,5-trisphosphate (PIP3).  In the appropriate cellular context, these two lipids can regulate a remarkably diverse array of physiological processes, including glucose homeostasis, cell growth, differentiation and motility.  These distinct functions are carried out by a family of eight related PI3-Ks in vertebrates that possess unique substrate specificities, localization, and modes of regulation.  Despite these known differences in upstream activation, the physiological roles of individual PI3-K isoforms remain largely unassigned, and dissecting the unique functions of members of this family is a major focus of ongoing research.  In addition to sequence homology within their catalytic domain, PI3-Ks share sensitivity to two small molecule inhibitors, wortmannin and LY294002.  Together, these two compounds have served as powerful probes for implicating PI3-Ks in a wide range of physiological processes, and much of our understanding of PI3-K action in cells derives from the use of these two reagents. Although wortmannin and LY294002 are broadly active against the PI3-K family, they show little specificity among PI3-K family members.   We have developed highly selective inhibitors of each individual PI3-K isoform (with a focus on p110&Beta;, p110, p110, p110and mTOR). Using these reagents, we have initiated a program to profile PI3-K isoforms and mTOR action in three physiological settings:  inflammatory signaling, glucose homeostasis, and cancer.  Several of the inhibitors we have developed are now in clinical trials (MLN0128, MLN1117, IPI-145).
**Neo-substrates as a new strategy for drug discovery:**  In an attempt to develop a drug to treat Parkinson's Disease we have developed a small molecule which helps to protect neurons from dying due to various types of stress. Mitochondria have long been implicated in the pathogenesis of Parkinson’s disease (PD). Mutations in the mitochondrial kinase PINK1 that reduce kinase activity are associated with mitochondrial defects and result in an autosomal-recessive form of early- onset PD. Therapeutic approaches for enhancing the activity of PINK1 have not been considered because no allosteric regulatory sites for PINK1 are known. We have shown that an alternative strategy, a neo-substrate approach involving the ATP analog kinetin triphosphate (KTP), can be used to increase the activity of both PD-related mutant PINK1G309D and PINK1WT.  Discovery of neo-substrates for kinases could provide a heretofore unappreciated modality for regulating kinase activity.**Drugging the most common oncogene:  K-Ras:**  In the area of cancer we have recently made a breakthrough by discovering a way to block the function of the GTPase, K-Ras.  Somatic mutations in the K-Ras are the most common activating lesions found in human cancer, and are generally associated with poor response to standard therapies. Efforts to directly target this oncogene have faced difficulties due to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites. We were able to develop small molecules that irreversibly bind to a common oncogenic mutant, K-Ras G12C. These compounds rely on the mutant cysteine for binding and therefore do not affect the wild type protein (WT). Using crystallography we identified a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region. Our results provide structure-based validation of a novel allosteric regulatory site on Ras that is targetable in a mutant-specific manner.